terasoluna-gfw-parent/pom.xml

```diff
@@ -177,6 +177,11 @@
                         </deployables>
                     </configuration>
                 </plugin>
+                <plugin>
+                    <groupId>org.codehaus.mojo</groupId>
+                    <artifactId>xml-maven-plugin</artifactId>
+                    <version>${xml-maven-plugin.version}</version>
```

Comment:
OK. I will delete  xml-maven-plugin setting on `terasoluna-gfw-parent`.As alternative solution, I will add below setting to pom.xml on `terasoluna-gfw` . 
This setting means to check xml format under terasoluna-gfw folder.  
```
        <plugin>
          <groupId>org.codehaus.mojo</groupId>
          <artifactId>xml-maven-plugin</artifactId>
          <version>${xml-maven-plugin.version}</version>
          <configuration>
            <indentSize>2</indentSize>
            <formatFileSets>
              <formatFileSet>
                <directory>${basedir}</directory>
                <includes>
                  <include>**/*.xml</include>
                </includes>
                <excludes>
                  <exclude>**/.*</exclude>
                </excludes>
              </formatFileSet>
            </formatFileSets>
            <useDefaultFormatFileSet>false</useDefaultFormatFileSet>
          </configuration>
```